core/src/main/java/ninja/soroosh/chatopia/core/runner/responses/Response.java

```diff
@@ -19,4 +19,20 @@ static PhotoResponse asPhoto(URL url) {
     static PhotoResponse asPhoto(String id) {
         return new PhotoResponse(id);
     }
+
+    static VideoResponse asVideo(InputStream inputStream) {
+        return new VideoResponse(inputStream);
+    }
+
+    static VideoResponse asVideo(URL url) {
+        return new VideoResponse(url);
+    }
+
+    static VideoResponse asVideo(String id) {
+        return new VideoResponse(id);
+    }
+
+    InputStream getStream();
```

Comment:
Why is this needed at the interface level?

Reply:
instead of using `getVideoStream()` I wanted to use something more abstract so in future it would support other messages as file.

Reply:
Right, but no every Response has stream, e.g. TextResponse